publikationsleitlinien.md

```diff
@@ -90,6 +91,11 @@ Werte mit Daten-Format `Uhrzeit` geben wir dem [internationalen Standard ISO 860
 ##### 5. URL 
 Werte mit Daten-Format `URL` schreiben wir standardmässig aufrufbar aus im Format `https://...`. <br>
 
+##### 6. Geografische Lage 
+Werte mit Daten-Format `Geokoordinaten` geben wir dem [globalen geodätischen Referenzsystem WGS 84](https://www.w3.org/TR/sdw-bp/) gemäss an als:
```

Comment:
D.h. es sind 2 Felder zu verwenden (latitude, longitude) und nicht nur eines? Ich würde das noch explizit erwähnen, da es auch möglich ist, die Werte in einem Feld abzulegen (z.B. komma-getrennt).

Reply:
Ich favorisiere 2 Felder. WDYT, @CGRBZH ?